### ISSUE TITLE
Evaluate button padding on main toolbar

### ISSUE THREAD

**Issue by SergeyZhukovsky** [brave/browser-android-tabs#296]
We can reduce space between icons on the URL bar and make the Edit control bigger. WDYT @jamesmudgett?
![pasted image at 2017_06_20 10_15 am](https://user-images.githubusercontent.com/12011303/27963427-03eed322-6303-11e7-8620-4c7054f9ce88.png)

**Comment by avently** [comment]
Can you change position of Brave icon? I think you can replace "info" button in menu. Or you can add all Brave's options to "Site settings". 
I'm asking you about that because now this icon is wasting space. I don't use it often. Only when some site is broken. Would be good to see a minimum number of icons in toolbar. 

P.S. couldn't do this myself. To much code involved into making this button work.

**Comment by SergeyZhukovsky** [comment]
Ya, probably is a good idea, but it is better to make it is an option, because you have lots of space on tablets for example and it doesn't consume the space.

**Comment by avently** [comment]
@SergeyZhukovsky yes, as option is good. I lost a day trying to figure out how to show this icon only when mTabSwitcherState == TAB_SWITCHER...
